jnigen/lib/src/logging/logging.dart

```diff
@@ -16,11 +18,9 @@ String _colorize(String message, String colorCode) {
   return message;
 }
 
-Logger log = Logger('jnigen');
-
-/// Set logging level to [level] and initialize the logger.
-void setLoggingLevel(Level level) {
-  Logger.root.level = level;
+Logger log = () {
```

Comment:
I did some lambda stuff here so that `log` is initialized to `INFO` even if `setLoggingLevel` is not called. This allows more parts to use `log`.

Reply:
sgtm